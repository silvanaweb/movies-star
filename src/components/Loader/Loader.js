import React from "react";
import "./Loader.css";

export const Loader = () => (
    <div className="Loader">
        <ul className="Loader__list">
            <li></li>
            <li></li>
            <li></li>
            <li></li>
            <li></li>
        </ul>
    </div>
)