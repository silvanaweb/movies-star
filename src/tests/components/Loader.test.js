import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import { Loader } from "../../components/Loader/Loader";

test("should render Loader correctly", () => {
  const renderer = new ShallowRenderer();
  const result = renderer.render(<Loader />);
  expect(result).toMatchSnapshot();
});
