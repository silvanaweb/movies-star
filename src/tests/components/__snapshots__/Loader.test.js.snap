// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Loader correctly 1`] = `
<div
  className="Loader"
>
  <ul
    className="Loader__list"
  >
    <li />
    <li />
    <li />
    <li />
    <li />
  </ul>
</div>
`;
